packages/custom-webpack/README.md

```diff
@@ -313,6 +313,25 @@ module.exports = (config, options) => {
 };
 ```
 
+Alternatively, using TypeScript :
+
+
+```typescript
+import { CustomWebpackBrowserSchema } from '@angular-builders/custom-webpack/dist';
```

Comment:
@just-jeb should this be used publicly?

Reply:
@arturovt  I don't see any problem with that.

Reply:
Maybe we could expose it publicly, thus it won’t be imported from the dist folder?

Reply:
Sure, up until TS support there was no need in exporting anything but now it totally makes sense.

Reply:
Actually it is already exported, so we just need to set `"types": "dist/index.d.ts"` in `package.json` (I believe it should work event without it, it should be the default entry)